packages/worker/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "worker",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "esbuild --bundle --platform=node --target=node20 --outfile=dist/index.js src/index.ts",
+    "package": "cd dist; mkdir -p ../target; zip -qr ../target/worker.zip ./*"
+  },
```

Comment:
It might be good to add a .gitignore under worker so ignore target contents. For api we created a seperate .gitignore too